src/lib/Makefile.am

```diff
@@ -20,7 +20,6 @@ librnp_la_SOURCES	= \
 	key_store.c \
 	key_store_pgp.c \
 	key_store_kbx.c \
-	key_store_ssh.c \
```

Comment:
I think @catap means, while we are not planning to generate SSH keys, we still need to access the SSH keychain to allow using PGP keys for SSH purposes, so we need the SSH keystore code?